src/versionbits.h

```diff
@@ -30,6 +30,8 @@ enum ThresholdState {
 // will either be NULL or a block with (height + 1) % Period() == 0.
 typedef std::map<const CBlockIndex*, ThresholdState> ThresholdConditionCache;
 
+extern const char * const VersionBitsDeploymentNames[];
```

Comment:
Whatever we're doing with C++11 in 0.13 doesn't help here, since this needs to be backported to 0.11 and 0.12.